pkg/agent/controller/networkpolicy/reconciler.go

```diff
@@ -398,11 +407,19 @@ func (r *reconciler) updateOFRule(ofID uint32, addedFrom []types.Address, addedT
 	return nil
 }
 
-func (r *reconciler) uninstallOFRule(ofID uint32) error {
+func (r *reconciler) uninstallOFRule(ofID uint32, flowDirection v1beta1.Direction) error {
```

Comment:
Here as well, please look at "Forget" function that calls uninstallOFRule.

Reply:
Thanks, I'll work on it.